gossip/src/cluster_info.rs

```diff
@@ -12,6 +12,13 @@
 //! * layer 2 - Everyone else, if layer 1 is `2^10`, layer 2 should be able to fit `2^20` number of nodes.
 //!
 //! Bank needs to provide an interface for us to query the stake weight
+
+#[deprecated(
+    since = "1.10.0",
```

Comment:
typically we use the current version of the oldest intended backport branch, so `1.10.6` in this case
```suggestion
    since = "1.10.6",
```

Reply:
Done